types/components/FormFileLabel.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+import { BsPrefixComponent } from './helpers';
+
+export interface FormFileLabelProps {
+  htmlFor?: string;
+  innerRef?: React.LegacyRef<this>;
```

Comment:
Same comment as above.

Reply:
Probably no reason at all. Left from FormCheck that I used as a basis for this. Removing since refs seems to be working without it.